intermediate_source/rpc_param_server_tutorial.rst

```diff
@@ -0,0 +1,398 @@
+
+Implementing a Parameter Server Using Distributed RPC Framework
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Author**\ : `Rohan Varma <https://github.com/rohan-varma>`_
+
+This tutorials walks through a simple example of implementing a [parameter server] using PyTorch's `Distributed RPC framework <https://pytorch.org/docs/stable/rpc.html>`__. This is a framework in which a set of servers store parameters, such as large embedding tables, and several trainers query the parameter servers in order to retrieve the most up to date parameters. These trainers can run a training loop locally and ocassionally synchronize with the parameter server to get the latest parameters. For more reading on the parameter server approach, check out `this paper <https://www.cs.cmu.edu/~muli/file/parameter_server_osdi14.pdf>`_.
```

Comment:
Is `[parameter server]` intended to be a link?

Reply:
Yes, although we link it a few sentences later, so I think it's fine to not link it here.